service/dap/server.go

```diff
@@ -486,19 +500,11 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 		s.binaryToRemove = debugname
 	}
 
-	// TODO(polina): support "remote" mode
-	if mode != "exec" && mode != "debug" && mode != "test" {
-		s.sendErrorResponse(request.Request,
-			FailedToLaunch, "Failed to launch",
-			fmt.Sprintf("Unsupported 'mode' value %q in debug configuration.", mode))
-		return
-	}
-
 	s.setLaunchAttachArgs(request)
 
 	var targetArgs []string
-	args, ok := request.Arguments["args"]
-	if ok {
+	args, modeValOk := request.Arguments["args"]
```

Comment:
What's wrong with 'ok' (here and above)?  Isn't that a fairly standard go idiom that also follows the style preference for shorter names? I guess we could change it to `present` for just map lookup to be more consistent with https://golang.org/doc/effective_go, but for type conversion `ok` should definitely be ok. Also, I am not sure how `modeVal` part helps readability when you are not actually dealing with `mode`.